packages/api/src/middleware/guards/userRights.guard.ts

```diff
@@ -55,10 +55,21 @@ export const canGetContact = async (
   } else if (req.user.status === UserStatus.PRO) {
     userRight = await getUserRightsById({
       _id: new mongoose.Types.ObjectId(req.params.userRightObjectId),
-      role: { $in: USER_ROLES_FOR_EDITION },
-      status: UserRightStatus.VERIFIED,
-      user: req.user._id,
     });
+
+    if (userRight) {
+      const placeRight = await getUserRightsById({
+        organization_id: userRight.organization_id,
+        place_id: userRight.place_id,
+        user: req.user._id,
+        role: { $in: USER_ROLES_FOR_EDITION },
+        status: UserRightStatus.VERIFIED,
+      });
+
+      if (!placeRight) {
+        userRight = null;
```

Comment:
J'aurais directement fait :
`return res.status(403).send({ message: "FORBIDDEN_USER" });`
Plutôt que d'assigner userRight à null pour invérifier la condition sous-jacente

Reply:
Assigner null à userRight au lieu de retourner immédiatement permet de garder un seul point de sortie, d'éviter la duplication de code et de rendre la logique plus claire

Reply:
Je vais être relou, mais j'ai l'impression que ce fix ne marchera pas. Ici, on défini placeRight pour déterminer si le compte pro actuel a les permissions sur la structure de sa propre orga, si ce n'est pas le cas, alors on indéfini userRights et on lui indique qu'il n'a pas les permissions.

Pour permettre qu'un pro puisse ajouter un pro, il faudrait changer ses permissions, pas lui retirer, ou je me gourre complêtement, mais ça me titille

```suggestion
        userRight = null;
    } else {
        userRight = placeRight;
    };
```